persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/impl/table/TableManager.java

```diff
@@ -48,6 +48,8 @@ default void dropTables(Connection conn) throws PersistenceException {
 
    String getDeleteRowSql();
```

Comment:
I think we can remove this now as it's not used anywhere else.

Reply:
Sure